### PR TITLE
Update rootless shortcomings with cgroup V2 information

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes:https://github.com/containers/libpod/issues/3778

**Comment by mheon** [comment]
LGTM

File: rootless.md [hunk 1]
```diff
@@ -6,18 +6,19 @@ Contributors are more than welcomed to help with this work.  If you decide to ca
...
+  * You can modify the `net.ipv4.ip_unprivileged_port_start` sysctl to change the lowest port.  For example `sysctl net.ipv4.ip_unprivileged_port_start=443` allows rootless Podman containers to bind to ports >= 443.
 * “How To” documentation is patchy at best.
 * If /etc/subuid and /etc/subgid are not setup for a user, then podman commands
 can easily fail
   * This can be a big issue on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
   * We are working to get support for NSSWITCH on the /etc/subuid and /etc/subgid files.
 * No cgroup V1 Support
   * cgroup V1 does not safely support cgroup delegation.
-  * However, cgroup V2 provides cgroup delegation and is available on Fedora starting with version 29 and other Linux distributions.
+  * As of Fedora 31 we have moved to cgroup V2 which has full support of rootless cgroup management
```
Comment (AkihiroSuda) [inline rootless.md]: `--cgroup-manager=systemd` should be mentioned here?

**Comment by TomSweeneyRedHat** [comment]
LGTM other than @AkihiroSuda question

**Comment by rhatdan** [comment]
Updated with comment on cgroup-manager for @AkihiroSuda

**Comment by rhatdan** [comment]
@mheon @AkihiroSuda @giuseppe @vrothberg @jwhonce @baude @QiWang19 PTAL

File: rootless.md [hunk 2]
```diff
@@ -6,18 +6,19 @@ Contributors are more than welcomed to help with this work.  If you decide to ca
...
 * If /etc/subuid and /etc/subgid are not setup for a user, then podman commands
 can easily fail
   * This can be a big issue on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
   * We are working to get support for NSSWITCH on the /etc/subuid and /etc/subgid files.
 * No cgroup V1 Support
   * cgroup V1 does not safely support cgroup delegation.
-  * However, cgroup V2 provides cgroup delegation and is available on Fedora starting with version 29 and other Linux distributions.
-* Some systemd's unit configuration options do not work in the rootless container
-  * systemd fails to apply several options and failures are silently ignored (e.g. CPUShares, MemoryLimit).
+  * As of Fedora 31 defaults to cgroup V2, which has full support of rootless cgroup management.  Note this requires the --cgroup-manager within rootless containers to use systemd, which new containers will get by default.
```
Comment (giuseppe) [inline rootless.md]: not sure it is worth adding but the devices controller is still not usable.  It requires eBPF that is a privileged operation

Comment (rhatdan) [reply rootless.md]: Could you open a separate PR on this.

**Comment by TomSweeneyRedHat** [comment]
Looks like some test unhappiness here.

**Pull request merged**